src/mutable_data.rs

```diff
@@ -354,26 +363,27 @@ impl UnseqMutableData {
         }
     }
 
-    /// Returns a value for the given key
+    /// Returns a value for the given key.
     pub fn get(&self, key: &[u8]) -> Option<&Vec<u8>> {
         self.data.get(key)
     }
 
-    /// Returns values of all entries
+    /// Returns values of all entries.
     pub fn values(&self) -> Vec<Vec<u8>> {
         self.data.values().cloned().collect()
     }
 
-    /// Returns all entries
+    /// Returns all entries.
     pub fn entries(&self) -> &UnseqEntries {
         &self.data
     }
 
-    /// Removes and returns all entries
+    /// Removes and returns all entries.
     pub fn take_entries(&mut self) -> UnseqEntries {
         mem::replace(&mut self.data, BTreeMap::new())
     }
 
+    /// Mutates entries based on `actions` for the provided user.
```

Comment:
Please add that it returns `Err(InvalidEntryActions)` in case if the mutation parameters are invalid.